pkgs/development/python-modules/requests-hawk/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi, python, mohawk, requests }:
+
+with lib;
+buildPythonPackage rec {
+  pname = "requests-hawk";
+  version = "1.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1qcga289yr6qlkmc6fjk0ia6l5cg0galklpdzpslf1y8ky9zb7rl";
+  };
+
+  propagatedBuildInputs = [ mohawk requests ];
+
+  meta = {
+    description = "Hawk authentication strategy for the requests python library.";
+    homepage = "https://github.com/sam-washington/requests-hawk";
+    license = licenses.asl20;
+    maintainers = [ ];
```

Comment:
please add yourself as maintainer

Reply:
Same conversation as last week :smile: : https://github.com/NixOS/nixpkgs/pull/98336#discussion_r491759482.

Reply:
There's over 3000 python packages in nixpkgs, it's nice to have other have context on a package.

Reply:
Done!